### PR TITLE
Fix TcpServer Example

### LINKED ISSUE #159: Error when running TcpServer example
When I try to run TcpServer example with the command:
```
sbt "examples/runMain zio.keeper.example.TcpServer"
```
I got an error:
```
Failed to instantiate SLF4J LoggerFactory
Reported exception:
java.lang.NoClassDefFoundError: ch/qos/logback/core/joran/spi/JoranException
	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:412)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:357)
	at zio.logging.slf4j.Slf4jLogger.$anonfun$logger$1(Slf4jLogger.scala:18)
	at zio.internal.FiberContext.evaluateNow(FiberContext.scala:333)
	at zio.internal.FiberContext.$anonfun$fork$2(FiberContext.scala:655)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: ch.qos.logback.core.joran.spi.JoranException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at sbt.internal.ManagedClassLoader.findClass(ManagedClassLoader.java:98)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 10 more
```

Looks like logger lib is missing, but on the pther hand I can start Node1 without any problems and it logs.

### PR DESCRIPTION
**Pull request by vpavkin**
Fixes #159.

Should merge just fine with #188

### CODE REVIEW

**Comment by mijicd** [comment]
Thanks for your contribution! It looks good to me, however, it will be obsoleted by #176, hence summoning @pshemass to sign it of.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 target
 .sbtopts
 project/.sbt
+.idea
```
Comment (pshemass) [inline .gitignore]: please follow instruction from this stackoverflow post

**Comment by pshemass** [comment]
> Thanks for your contribution! It looks good to me, however, it will be obsoleted by #176, hence summoning @pshemass to sign it of.

@mijicd most of the stuff will be obsoleted by #176 but we need to break it down my refactor anyway so let merge this I will deal with conflicts.

File: .gitignore (hunk 1 above)
Comment (vpavkin) [reply .gitignore]: fixed

**Pull request merged**